pkg/utils/gardener/shoot.go

```diff
@@ -185,6 +187,7 @@ func GetShootProjectSecretSuffixes() []string {
 		ShootProjectSecretSuffixSSHKeypair,
 		ShootProjectSecretSuffixOldSSHKeypair,
 		ShootProjectSecretSuffixMonitoring,
+		ShootProjectSecretSuffixMonitoringOperator,
```

Comment:
https://github.com/gardener/gardener/blob/24dc519d70c664bb64ebe46808320ae9595d3cfc/pkg/utils/gardener/shoot_test.go#L286 
Should be updated